crypto/evp/evp_lib.c

```diff
@@ -404,7 +404,7 @@ int EVP_MD_meth_get_app_datasize(const EVP_MD *md)
 }
 unsigned long EVP_MD_meth_get_flags(const EVP_MD *md)
 {
-    return md->block_size;
+    return md->flags;
```

Comment:
... enough that it deserves a PR of its own, even if that seems a bit silly.  Will you or shall I steal it from you?